feature/mpls/sr/otg_tests/isis_node_sid_forward/README.md

```diff
@@ -0,0 +1,61 @@
+# SR-1.1: Transit forwarding to Node-SID via ISIS
+
+## Summary
+
+MPLS-SR transit forwarding to Node-SID distributed over ISIS
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Configuration
+
+Topology: ATE1—DUT1–ATE2
+                               
+*   Configure Segment Routing Global Block (start: 400000 range: 65001)
+*   Enable Segment Routing for the ISIS
+*   Enable MPLS forwarding
+
+*  Prefix (1) with node-SID is advertised by the direct ISIS neighbor
+*  Prefix (2) with node-SID is advertised by simulated indirect ISIS speaker
+
+### Test 
+
+Verify that:
+
+*  DUT receives prefixes with SIDs and has a correct emulated ISIS topology (database).
+*  DUT advertises both prefixes with node-SID to ATE2.
+
+Generate traffic:
+*   Send labeled traffic transiting through the DUT matching direct prefix (1). Verify that ATE2 receives traffic with node-SID label popped.
+*   Send labeled traffic transiting through the DUT matching indirect prefix (2). Verify that ATE2 receives traffic with the node-SID label intact.
+*   Verify that corresponding SID forwarding counters are incremented.
+*   Traffic arrives without packet loss.
+
+## OpenConfig Path and RPC Coverage
+
+```yaml
+paths:
+  # configuration
+  /network-instances/network-instance/mpls/global/interface-attributes/interface/config/mpls-enabled:
```

Comment:
Should these paths also be claimed as covered by this test since a global label block is configured?

  /network-instances/network-instance/mpls/global/reserved-label-blocks/reserved-label-block/config/local-id:
  /network-instances/network-instance/mpls/global/reserved-label-blocks/reserved-label-block/config/lower-bound:
  /network-instances/network-instance/mpls/global/reserved-label-blocks/reserved-label-block/config/upper-bound:

Reply:
Added paths.